core/services/feeds/orm_test.go

```diff
@@ -441,6 +447,75 @@ func Test_ORM_ApproveJobProposal(t *testing.T) {
 	assert.Equal(t, actualCreated.ProposedAt, actual.ProposedAt)
 }
 
+func Test_ORM_CancelJobProposal(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	orm := setupORM(t)
+	fmID := createFeedsManager(t, orm)
+	externalJobID := uuid.NullUUID{UUID: uuid.NewV4(), Valid: true}
+
+	jp := &feeds.JobProposal{
+		RemoteUUID:     uuid.NewV4(),
+		Spec:           "",
+		Status:         feeds.JobProposalStatusPending,
+		FeedsManagerID: fmID,
+	}
+
+	// Defer the FK requirement of a job proposal.
```

Comment:
Yes this is obvious, might be more helpful if the comment explained _why_ ;)

Reply:
Updated :D